.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,100 @@
+name: Publish Docs to Github Pages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  docs-agentkit:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cdp-agentkit-core
+    strategy:
+      matrix:
+        python: ['3.10']
+
+    steps:
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
```

Comment:
We don't need to add complexity with the matrix since we will only use one version. Here is an example with single version SDK repo. https://github.com/coinbase/cdp-sdk-python/blob/be5d376f068f0be59350d56bf04ef643674834bd/.github/workflows/publish_docs.yml#L17

cc: @stat